beanstalkworker/rawJob.go

```diff
@@ -88,6 +88,11 @@ func (job *RawJob) GetTube() string {
 	return job.tube
 }
 
+// GetConn returns the beanstalk connection
```

Comment:
@NevilleC can you also update the demo app to use this function in some way - that way the demo can use all of the features.